.github/workflows/unit-tests.yml

```diff
@@ -77,6 +77,7 @@ jobs:
           docker exec clickhouse clickhouse-client --query "CREATE SETTINGS PROFILE clickhouse_settings SETTINGS flatten_nested=0, join_use_nulls=1 TO clickhouse_test, clickhouse_read_only"
           docker exec clickhouse clickhouse-client --query "GRANT SELECT, SYSTEM VIEWS, dictGet ON *.* TO clickhouse_read_only"
           docker exec clickhouse clickhouse-client --query "CREATE NAMED COLLECTION seqr_postgres_named_collection AS user='postgres', password='pgtest', host='postgres', port='5432'"
+          docker exec clickhouse clickhouse-client --query "CREATE NAMED COLLECTION pipeline_data_access AS access_key_id='123', secret_access_key='456'"
```

Comment:
In our open source helm template, this collection only exists for users using gs:
https://github.com/broadinstitute/seqr-helm/blob/0891f02e41b0b3f129bb69aaee08d999782586d0/charts/seqr/templates/clickhouseconfig.yaml#L35

Will this be a problem for local users not using gs?

Reply:
I think since hgmd does not need to work for local installs its okay to leave the configuration/ env vars as is unless you feel strongly. I would say we should take this out of unit test set up if its not needed though